src/Controllers/SettingsController.php

```diff
@@ -39,10 +107,27 @@ public function oauth(): Response
         }
 
         return $this->response->withView(
-            'pages/settings/oauth.twig',
+            'pages/settings/oauth',
             [
+                'settings_menu' => $this->settingsMenu(),
                 'providers' => $providers,
             ] + $this->getNotifications(),
         );
     }
+
+    /**
+     * @return array
+     */
+    public function settingsMenu(): array
+    {
+        $menu = [
+            url('/user-settings') => 'settings.profile',
```

Comment:
This should be moved to `additional.po` as it has no direct translation reference